src/openforms/registrations/contrib/objects_api/plugin.py

```diff
@@ -66,17 +66,20 @@ class ObjectsAPIRegistration(BasePlugin):
         ),
     }
 
+    @override
```

Comment:
Thoughts on the `override` decorator? When dealing with plugins I often get confused on which method is actually an implementation of the base class and which one isn't.

It's a bit useless in this case, as every method is an override :confused:

Reply:
In my editor I generally see a little icon on the left of the method which indicates if this method overrides a method of a parent and if it is overwritten by a child (or both). It is very handy